akka-persistence/src/main/scala/akka/persistence/journal/EventAdapters.scala

```diff
@@ -16,8 +16,10 @@ import scala.collection.mutable.ArrayBuffer
 import scala.reflect.ClassTag
 import scala.util.Try
 
-/** INTERNAL API */
-private[akka] class EventAdapters(
+/**
+ * `EventAdapters` serves as a per-journal collection of bound event adapters.
+ */
+class EventAdapters(
```

Comment:
This change is needed to make the public `Persistence(system).adaptersFor` meaningful.